scheduler/src/cook/components.clj

```diff
@@ -436,7 +434,20 @@
                      (when enabled?
                        (when (zero? port)
                          (throw (ex-info "You enabled nrepl but didn't configure a port. Please configure a port in your config file." {})))
-                       ((lazy-load-var 'clojure.tools.nrepl.server/start-server) :port port)))}))
+                       ((lazy-load-var 'clojure.tools.nrepl.server/start-server) :port port)))
+     :retrieve-url-path-fn (fnk [[:config {agent-query-cache {:threshold 1000, :ttl 6000}}]]
+                             (let [{:keys [threshold ttl]} agent-query-cache
+                                   _ (log/info "Agent query cache will be created using:" {:threshold threshold, :ttl ttl})
```

Comment:
We are ignoring any other config present in the `agent-query-cache`, so in the presence of other fields that log statement would be misleading.